ganttproject/src/main/java/net/sourceforge/ganttproject/calendar/CalendarEditorPanel.java

```diff
@@ -470,4 +486,58 @@ public void setValueAt(Object aValue, int row, int col) {
       }
     }
   }
+
+  private static class CalendarAction extends GPAction {
+    private  TableModelImpl myTableModel;
+
+    public CalendarAction(TableModelImpl tableModel) {
+        super("Calendar"); // Need to localize
+      
+        myTableModel = tableModel;
+        
+        // Don't know how to set ENABLE for the action
+        //
+        putValue(AbstractTableAndActionsComponent.PROPERTY_IS_ENABLED_FUNCTION,
+          (Function<List<T>, Boolean>) input -> {
+            return true;
+        });
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+      Platform.runLater(new Runnable() {
+        @Override
+        public void run() {
+
+          javafx.scene.control.Dialog dialog = new javafx.scene.control.Dialog();
+          dialog.setTitle("Calendar");
+
+          DialogPane dialogPane = dialog.getDialogPane();
+
+          MultiDatePicker multiDatePicker = new MultiDatePicker();
+          multiDatePicker.setValue(LocalDate.now());
+
+          dialogPane.setContent(multiDatePicker.getPopupContent());
+          dialogPane.getButtonTypes().add(ButtonType.APPLY);
+
+          Optional result = dialog.showAndWait();
+          
+          addDatesToTable(multiDatePicker.getSelectedDates());
+        }
+
+        private void addDatesToTable(List<LocalDate> selectedDates) {
+
+          // TableModelImpl returns wrong row count (+1)
+          //
+          int rowCount = myTableModel.getRowCount() - 1;
+
+          for (int i=0; i<selectedDates.size(); i++) {
```

Comment:
formatting nitpicking: please put spaces around `=` and `<`, use two spaces indent

Reply:
Done